x/auth/module.go

```diff
@@ -216,8 +219,27 @@ func ProvideModule(in AuthInputs) AuthOutputs {
 		maccPerms[permission.Account] = permission.Permissions
 	}
 
-	k := keeper.NewAccountKeeper(in.Cdc, in.Key, types.ProtoBaseAccount, maccPerms, in.Config.Bech32Prefix, types.NewModuleAddress(govtypes.ModuleName).String())
-	m := NewAppModule(in.Cdc, k, simulation.RandomGenesisAccounts, in.LegacySubspace)
+	// default to governance authority if not provided
+	authority := types.NewModuleAddress(govtypes.ModuleName)
```

Comment:
This boilerplate in present in all modules, but I am not sure what is the best place to extract it. It is pretty specific and any package extracting this package will have a dependency on auth. Is it ok to have `types` to depend on `x/auth`?

Reply:
@aaronc about the code duplication, what package is the best?

Reply:
Mmh, actually auth seems like a good place, wdyt?